### ISSUE TITLE
task=track

### ISSUE THREAD

**Issue by JohnAsath** [ultralytics/ultralytics#18955]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

1.  !yolo task=detect mode=train model=yolov8m.pt data=/content/drive/MyDrive/Tuck-in_Tuck-out/data.yaml epochs=300 imgsz=1280 batch=8 patience=10 project=/content/drive/MyDrive/Tuck-in_Tuck-out/results name=test

instead of task = detect, task=track what will happen?

2.  How should the data be formatted for tracking, and what should the TXT format be?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @JohnAsath, thank you for your interest in Ultralytics 🚀! We recommend exploring our [Docs](https://docs.ultralytics.com/) where you can find detailed guidance on various tasks, including detection and tracking, as well as [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples. Many of the most common questions are addressed there!

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training or tracking ❓ Question, please include as much information as possible about your setup, such as your training command, how your data is structured, and any logs or errors you are encountering. For tracking-specific tasks, ensure your data adheres to the [tracking dataset format](https://docs.ultralytics.com/tasks/tracking/).

For assistance with tracking-related questions specifically:
1. Switching from `task=detect` to `task=track` changes the behavior to track object movements across frames. See the [Tracking Guide](https://docs.ultralytics.com/tasks/tracking/) for more information.
2. Ensure your data is formatted correctly for tracking. Refer to the required [TXT annotation structure](https://docs.ultralytics.com/tasks/tracking/#track-dataset-format) and the [MOT-compliant dataset details](https://docs.ultralytics.com/tasks/tracking/#format-requirements).

Join the Ultralytics community where it suits you best for more help and discussions. For real-time support, check out [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer more in-depth conversations? Visit our [Discourse](https://community.ultralytics.com/). Or share knowledge and questions with others on our [Subreddit](https://reddit.com/r/Ultralytics) 🌟.

## Upgrade

Upgrade to the latest `ultralytics` package to ensure you’re working with the most up-to-date features and fixes. Install it in a [**Python>=3.8**](https://www.python.org/) environment with:

```bash
pip install -U ultralytics
```

## Environments

YOLO, including tracking tasks, runs seamlessly in the following environments with dependencies (e.g., [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), [PyTorch](https://pytorch.org/)) pre-installed:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests ensure proper operation of YOLO for all [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

This is an automated response to help guide you 👾. An Ultralytics engineer will review your issue and provide personalized assistance as soon as possible. Stay tuned! 😊

**Comment by Y-T-G** [comment]
1. Nothing probably. Or will default to `detect` based on the model being trained.
2. There's no support for training using tracking datasets

**Comment by guguqi123** [comment]
Hello, may I ask how to only display the ID and not show the class labels when using YOLO's built-in BoT-SORT for tracking?
